client/state/partner-portal/licenses/hooks/use-products-query.ts

```diff
@@ -44,29 +45,23 @@ function queryProducts(): Promise< APIProductFamily[] > {
 		} );
 }
 
-export default function useProductsQuery< TError = unknown, TData = unknown >(
-	options?: UseQueryOptions< APIProductFamily[], TError, TData >
-): UseQueryResult< TData, TError > {
+export default function useProductsQuery(): UseQueryResult< APIProductFamilyProduct[], unknown > {
```

Comment:
My changes to this hook were originally made because I began storing the products in redux, but after feedback in p9oQ9f-1qT-p2 I changed how that worked. However I left these changes because I think they make the use of the hook more intuitive and consistent.